page/learningchoice.jsx

```diff
@@ -24,18 +28,66 @@ css(`
 }
 `)
 
-export const LearningChoice = () => (
-  <Div class="div-learning">
-    <P class="title-learning">What do you want to learn 📚 ?</P>
-    {Object.entries(roles).map(([key, { name, color }]) => (
-      <span class="navlink-learning" key={key}>
-        <NavLink
-          href={`${API}/link/discord?speciality=${key}`}
-          style={{ color: parseColor(color), fontSize: 35 }}
+export const LearningChoice = () => {
+  let [errors, setErrors] = useState({})
+  if (!user) {
+    navigate('/login')
+    return
+  }
+  const send = (e) => {
+    e.preventDefault()
+    const form = new FormData(e.target)
+    const { why, speciality } = Object.fromEntries(form)
+    if (!why) {
+      setErrors((d) => {
+        return { ...d, why: 'you must fill this input' }
+      })
+      return
+    }
+    location.href = `${API}/link/discord?speciality=${speciality}`
+  }
+
+  return (
+    <Div class="div-learning">
+      <Form
+        submit="Go"
+        onSubmit={send}
+        style={{ textAlign: 'center' }}
+        title="Formulaire"
+      >
+        <Input
+          name="why"
+          comment="why do you want to join NaN?"
+          errors={errors}
+          type="text"
+        />
+        <br />
+        <br />
+        <Select
+          name="speciality"
+          comment="why do you want to learn 📚 ?"
```

Comment:
i think that the correct sentence must be `What do you want to learn` instead of `why do you want to learn` for the select input

Reply:
yes, what do you want to learn, thank you